tailwind.config.js

```diff
@@ -14,6 +14,26 @@ module.exports = {
         xl: "1200px",
         "2xl": "1400px",
       },
+      fontFamily: {
+        "inter-medium": ["Inter-Medium"],
```

Comment:
If we are using `Inter-Medium` only then let's delete other font files.  We will gradually add files based on the requirements